src/config/server.rs

```diff
@@ -87,6 +87,9 @@ pub struct Server {
     pub html_render_cache_max_capacity: u64,
 
     pub content_security_policy: Option<HeaderValue>,
+
+    pub docs_rs_base_url: url::Url,
+    pub docs_rs_api_token: String,
```

Comment:
since we only need this for the initial setup and then won't touch the values in the config anymore I'm wondering about having a `RealDocsRsClient::from_environment() -> Option<Self>` that can be used in the `bin/background-worker.rs` file. That would allow us to completely remove the fields from the config and simplify the setup a little bit.